<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1194</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Lucida Calligraphy</family>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <widget class="QLabel" name="status">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>190</y>
          <width>291</width>
          <height>71</height>
         </rect>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string> Main Window</string>
       </attribute>
       <widget class="QLCDNumber" name="lcdNumber">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>480</y>
          <width>64</width>
          <height>23</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>481</width>
          <height>101</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Panel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <property name="midLineWidth">
         <number>5</number>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600; font-style:italic;&quot;&gt;Welcome to the College Tour Program&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>950</x>
          <y>10</y>
          <width>258</width>
          <height>501</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="display_college_button">
           <property name="text">
            <string>Display Colleges</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_4">
           <property name="text">
            <string>Display Souvenirs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_3">
           <property name="text">
            <string>Find Distance</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButton_5">
           <property name="text">
            <string>Find Souvenirs</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Display Campuses</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget">
        <property name="geometry">
         <rect>
          <x>18</x>
          <y>10</y>
          <width>911</width>
          <height>511</height>
         </rect>
        </property>
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>2</number>
        </property>
       </widget>
       <widget class="QPushButton" name="display_college_list_button">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>280</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Display</string>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>490</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>960</x>
          <y>430</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Comic Sans MS</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
       <widget class="QWidget" name="">
        <property name="geometry">
         <rect>
          <x>930</x>
          <y>180</y>
          <width>225</width>
          <height>81</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QRadioButton" name="display_souvenirs_radio_button">
           <property name="font">
            <font>
             <family>Lucida Calligraphy</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Display Souvenirs</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="display_colleges_radio_button">
           <property name="font">
            <font>
             <family>Lucida Calligraphy</family>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="text">
            <string>Display Colleges</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Search Campuses</string>
       </attribute>
       <widget class="QTableWidget" name="tableWidget_2">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>20</y>
          <width>741</width>
          <height>71</height>
         </rect>
        </property>
       </widget>
       <widget class="QProgressBar" name="progressBar_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>490</y>
          <width>161</width>
          <height>23</height>
         </rect>
        </property>
        <property name="value">
         <number>1</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="textDirection">
         <enum>QProgressBar::TopToBottom</enum>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_2">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>430</y>
          <width>171</width>
          <height>51</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Lucida Calligraphy</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_3">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>300</y>
          <width>741</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableWidget_4">
        <property name="geometry">
         <rect>
          <x>410</x>
          <y>110</y>
          <width>741</width>
          <height>171</height>
         </rect>
        </property>
       </widget>
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>10</y>
          <width>20</width>
          <height>501</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>19</y>
          <width>361</width>
          <height>221</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="pushButton_6">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>170</y>
           <width>111</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Lucida Calligraphy</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>275</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Campus ID</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="label_4">
         <property name="geometry">
          <rect>
           <x>140</x>
           <y>70</y>
           <width>55</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>OR</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>298</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Campus Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>259</y>
          <width>361</width>
          <height>161</height>
         </rect>
        </property>
        <property name="title">
         <string/>
        </property>
        <widget class="QPushButton" name="pushButton_7">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>101</width>
           <height>31</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Lucida Calligraphy</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Find</string>
         </property>
        </widget>
        <widget class="QWidget" name="">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>10</y>
           <width>341</width>
           <height>25</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Distance From</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox"/>
          </item>
         </layout>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1194</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>Info</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuExit">
    <property name="title">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuAdmin">
    <property name="title">
     <string>Admin</string>
    </property>
   </widget>
   <addaction name="menuInfo"/>
   <addaction name="menuExit"/>
   <addaction name="menuAdmin"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
